src/explore-education-statistics-common/src/modules/table-tool/components/TableSizeError.tsx

```diff
@@ -0,0 +1,77 @@
+import Button from '@common/components/Button';
+import ErrorPrefixPageTitle from '@common/components//ErrorPrefixPageTitle';
+import downloadService from '@common/services/downloadService';
+import { Subject } from '@common/services/tableBuilderService';
+import React, { useEffect, useRef } from 'react';
+
+interface Props {
+  focus: boolean;
+  releaseId?: string;
+  showDownloadOption?: boolean;
+  subject: Subject;
+}
+
+const TableSizeError = ({
+  focus = false,
+  releaseId,
+  showDownloadOption = true,
+  subject,
+}: Props) => {
+  const handleDownloadFile = async (fileId: string) => {
+    if (releaseId) {
+      await downloadService.downloadFiles(releaseId, [fileId]);
+    }
+  };
+
+  const ref = useRef<HTMLDivElement>(null);
+
+  useEffect(() => {
+    if (focus && ref.current) {
+      ref.current.focus();
+    }
+  }, [focus]);
+
+  return (
+    <div
+      aria-labelledby="tableSizeError"
+      className="govuk-error-summary"
+      id="filtersForm-tableSizeError"
+      ref={ref}
+      role="alert"
+      tabIndex={-1}
+    >
+      <h2 className="govuk-error-summary__title" id="tableSizeError">
+        There is a problem
+      </h2>
+
+      <ErrorPrefixPageTitle />
+      <div className="govuk-error-summary__body">
+        <p>
+          A table cannot be returned as the filters chosen can exceed the
+          maximum allowable table size.
```

Comment:
This error message is a little weird - would suggest rewording it to 'Could not create table as the filters chosen may exceed the maximum allowed table size'

Reply:
Happy that this has suggestion has been addressed